assets/styles/_search-results.scss

```diff
@@ -69,6 +69,12 @@
     margin: 0;
     list-style: none;
   }
+
+  .info {
```

Comment:
The `info` name is too generic IMHO, so I would prefer something like `search-results-sort-order` or similar.